Gemfile

```diff
@@ -16,6 +16,8 @@ gem "uglifier", "~> 4.2"
 
 group :development, :test do
   gem "govuk_test", "~> 1.0"
+  gem "jasmine", "~> 3.5.1"
+  gem "jasmine_selenium_runner", "~> 3", require: false
```

Comment:
Sorry to be a pain again, we pushed up a new approach to pulling in jasmine_selenium_runner that avoids the need for `require: false` that we're going to try popularise before the `require: false` technique spreads too far (as it essentially fights the gems configuration). Details are here: https://github.com/alphagov/content-publisher/pull/2110